src/main/java/core/basesyntax/AppMain.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitOperationDao;
+import core.basesyntax.dao.FruitOperationDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitOperation;
+import core.basesyntax.report.convertdata.DataConvertor;
+import core.basesyntax.report.convertdata.DataConvertorImpl;
+import core.basesyntax.report.input.FileReaderImpl;
+import core.basesyntax.report.input.Reader;
+import core.basesyntax.report.output.FileWriterImpl;
+import core.basesyntax.report.output.Writer;
+import core.basesyntax.report.report.ReportGenerator;
+import core.basesyntax.report.report.ReportGeneratorImpl;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.operation.BalanceOperationHandler;
+import core.basesyntax.strategy.operation.OperationHandler;
+import core.basesyntax.strategy.operation.PurchaseOperationHandler;
+import core.basesyntax.strategy.operation.ReturnOperationHandler;
+import core.basesyntax.strategy.operation.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AppMain {
+    // HINT: In the `public static void main(String[] args)`
+    // it is better to create instances of your classes,
+    // and call their methods, but do not write any business logic in the `main` method!
+    private static final String FILE_READ_SRC = "src/main/java/core/basesyntax/reportToRead.csv";
+    private static final String FILE_WRITE_SRC = "finalReport.csv";
```

Comment:
Place the input and output files into the `src/main/resources` folder, not in the `src/main/java`.